### ISSUE TITLE
Allow configurable symlink for .tmp/public

### ISSUE THREAD

**Issue by dbkaplun** [balderdashy/sails#3044]

**Comment by tjwebb** [comment]
Can you provide more info on what you're trying to achieve, here?

**Comment by dbkaplun** [comment]
My goal is to have Sails serve static files from `/` that do not interfere with routes. Sails currently does this by copying/compiling into `.tmp/public`, which is a fine solution. However some people might want to skip that whole step and have Sails serve a directory straight. It seems like if you remove all or parts of the grunt configuration, you can set up a symlink from `.tmp/public` to some other directory safely, which accomplishes my goal, but it's a fairly roundabout way of accomplishing this.

**Comment by sailsbot** [comment]
Thanks for posting, @dbkaplun.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
